docs/vsphere_ci/scripts/autounattend.xml

```diff
@@ -62,8 +62,11 @@
             </ImageInstall>
             <UserData>
                 <AcceptEula>true</AcceptEula>
+                <!-- Product Key from https://learn.microsoft.com/en-us/windows-server/get-started/kms-client-activation-keys#windows-server-2022 -->
+                <!-- This allows us to get around having to intervene and skip entering the key during the packer build -->
                 <ProductKey>
                     <Key>XXXXX-XXXXX-XXXXX-XXXXX-XXXXX</Key>
+                    <Key>VDYBN-27WPP-V4HQT-9VMD4-VMK7H</Key>
```

Comment:
The proposed  Product Key maps to `Windows Server 2022 Standard`. Should we use `Windows Server 2022 Datacenter`? See [Comparison of Standard and Datacenter editions of Windows Server 2022](https://learn.microsoft.com/en-us/windows-server/get-started/editions-comparison-windows-server-2022?tabs=full-comparison)

Reply:
It really doesn't matter as we are never going to activate this Windows instance.

Reply:
Makes sense.